plugins/parsers/json/parser.go

```diff
@@ -73,7 +76,14 @@ func (p *Parser) parseArray(data []interface{}) ([]telegraf.Metric, error) {
 		case map[string]interface{}:
 			metrics, err := p.parseObject(v)
 			if err != nil {
-				return nil, err
+				log.Printf("E! [parsers.json] Invalid JSON object: %v", v)
+
+				if p.strict {
+					return nil, err
+				}
+
+				log.Print("I! [parsers.json] Strict not enabled, skipping.")
```

Comment:
Debug level or remove, since we have selected non-strict mode this is not an error, so we wouldn't want to print it each interval.